pax_experiments/sequence_learning/sequence_experiments_150128.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Created on Thu Jan 29 16:18:44 2015
+
+This file runs sequence memory experiments. A set of random sequences are
+created from a library of SDRs. The sequences are presented to the temporal 
+memory in a random order, and it learns the sequences. Because of the limited
+dictionary of SDRs, most sequences created will be "high-order", in that 
+different sequences will contain the same pieces.
+
+@author: pfrady
```

Comment:
We don't put the @author tag. Over time most files will have multiple contributors. The git commit log will keep track of author for each commit.